src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -2051,6 +2890,22 @@ instruct vsqrtD(vReg dst, vReg src) %{
   ins_pipe(pipe_slow);
 %}
 
+// vector sqrt - predicated
+
+instruct vsqrt_masked(vReg dst_src, vRegMask_V0 v0) %{
```

Comment:
Maybe it's better to split predicated version into `vsqrtF_masked` and `vsqrtD_masked` just like vector sqrt did.